### PR TITLE
Tier2 - CephFS-NFS - CEPH-83575095

### PR DESCRIPTION
**Pull request by hkadam134**
# Description

CEPH-83575095 - Upgrade a stand-alone NFS Cluster to HA and verify all the exports are intact and functioning as before

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-A1EAMW/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_nfs/test_cephfs_upgrade_nfs_standalone_to_nfs_ha_cluster.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 8 --file-size 10240 "
+        f"--files 10 --top {mounting_dir}{dir_name}",
+    )
+
+
+@retry(CommandFailed, tries=5, delay=5)
+def validate_nfs_ha_server(client, nfs_mounting_dir, dir_name):
+    try:
+        client.exec_command(sudo=True, cmd=f"ls -l {nfs_mounting_dir}{dir_name}")
+        client.exec_command(sudo=True, cmd=f"rm -rf {nfs_mounting_dir}{dir_name}")
```
Comment (neha-gangadhar) [inline tests/cephfs/cephfs_nfs/test_cephfs_upgrade_nfs_standalone_to_nfs_ha_cluster.py:169]: Can we write IO to same directory post upgrade and validate? Is it necessary to delete and create which becomes new altogether.

**Comment by neha-gangadhar** [comment]
LGTM

**Review by AmarnatReddy — APPROVED** [review]
LGTM

File: tests/cephfs/cephfs_nfs/test_cephfs_upgrade_nfs_standalone_to_nfs_ha_cluster.py (hunk 1 above)
Comment (hkadam134) [reply tests/cephfs/cephfs_nfs/test_cephfs_upgrade_nfs_standalone_to_nfs_ha_cluster.py:169]: Just to validate the existing folder remains in HA setup, I am doing "ls" and removing an existing folder

**Pull request merged**